core/scripts/umip-3/3_Verify.js

```diff
@@ -35,12 +41,12 @@ async function runExport() {
   console.log(" 1. Validating deployed bytecode at new addresses...");
 
   // The deployed bytecode should match the expected bytecode for all new contracts deployed.
-  await compiledByteCodeMatchesDeployed(Voting);
-  await compiledByteCodeMatchesDeployed(Registry);
-  await compiledByteCodeMatchesDeployed(Store);
-  await compiledByteCodeMatchesDeployed(FinancialContractsAdmin);
-  await compiledByteCodeMatchesDeployed(IdentifierWhitelist);
-  await compiledByteCodeMatchesDeployed(Governor);
+  // await compiledByteCodeMatchesDeployed(Voting);
```

Comment:
Should we just get rid of this script now if we're not verifying bytecode?

Reply:
we can. I thought we could leave it for now as we might at some point want to verify this in some way.